src/main/java/com/tidal/wave/browser/Driver.java

```diff
@@ -8,12 +8,30 @@
 
 import java.util.Objects;
 
+/**
+ * Manages the creation and destruction of WebDriver instance. <br>
+ * The driver output types would be: <br>
+ * -- Chrome <br>
+ * -- Firefox <br>
+ * -- Edge <br>
+ * Safari creates its own browser instance. <br>
+ * IE is not supported.
+ *
+ */
 public class Driver {
     private final Logger logger = LoggerFactory.getLogger(Driver.class);
 
     private DriverCommand driverCommand;
 
-    public DriverCommand create(String browserType, AbstractDriverOptions<?> options) {
+    /**
+     * Creates the Driver(Browser) instance with the given parameters.
+     * Gets the driver from the stored data if it is already created.
+     *
+     * @param browserType Choose the browser type.
+     * @param options The browser options to be supplied.
+     * @return DriverCommand instance to navigate and terminate the browser sessions
+     */
+    protected DriverCommand create(String browserType, AbstractDriverOptions<?> options) {
```

Comment:
I had one instance where I was using this method directly... (in hindsight probably unnecessary from me) but I wonder if this will effect anyone else?

Reply:
I have made it 'protected' now. Only the same package classes can use it.